scripts/west_commands/runners/core.py

```diff
@@ -268,6 +268,9 @@ class RunnerCaps:
       connected to a single computer, in order to select which one will be used
       with the command provided.
 
+    - dev_ids: whether the runner supports multiple device identifiers for a
```

Comment:
Small remark, having both `dev_id` and `dev_ids` keeps it concise, but it might be confusing too. Not opposed to this, just thinking if an alternative name like `flash_multiple` is clearer? Users will still pass `dev-id` arguments, never `dev-ids`.

Reply:
Good idea! Will do.

Reply:
Done!